### PR TITLE
Update ASP.NET UnitTest section for RC2 changes

### PR DESCRIPTION
**Pull request by uxp**
There was some confusing example code within the Unit Test section of the aspnet documentation due to the changes introduced with RC2. This should hopefully clarify some of that.

### CODE REVIEW

**Comment by danroth27** [comment]
Looks good to me :shipit:

@ardalis Can you look this over as well?

**Comment by ardalis** [comment]
Looks good to me as well. :shipit:

File: aspnet/testing/unit-testing/sample/src/PrimeWeb/Startup.cs [hunk 1]
```diff
@@ -21,7 +21,6 @@ public void ConfigureServices(IServiceCollection services)
             IHostingEnvironment env)
         {
             // Add the platform handler to the request pipeline.
```
Comment (uxp) [inline aspnet/testing/unit-testing/sample/src/PrimeWeb/Startup.cs]: I need to remove this comment. It doesn't pertain to the line that follows.

**Comment by Rick-Anderson** [comment]
@uxp thanks. Merged. It's taking time to see the merge.

**Pull request closed without merging**